package.json

```diff
@@ -34,7 +34,7 @@
     "doctoc": "doctoc README.md packages/collector/README.md",
     "lint": "eslint .*.js *.js && npm run typecheck && lerna run lint --stream && eslint misc && eslint dockerfile-examples && eslint bin",
     "tekton:lint": "npx @ibm/tekton-lint .tekton/**/**/*.yaml .tekton/**/*.yaml .tekton/*.yaml",
-    "postinstall": "husky install && lerna run node_modules:exists",
+    "postinstall": "chmod +x ./node_modules/husky/lib/bin.js && husky install && lerna run node_modules:exists",
```

Comment:
I think we should migrate from v7 to v9. This might resolve the issue.
https://github.com/typicode/husky/releases/tag/v9.0.1

But not part of this PR. We can create a JIRA ticket.

Reply:
What specific error are we encountering with Husky? Is it occurring only in certain cases, such as with M1 devices? Any github issues already opened?

Reply:
I could not find any specific issues for M1 devices. The issue was that the husky binary file didn't have write permission to perform `husky install`.

Reply:
> I think we should migrate from husky v7 to v9. This might resolve the issue. https://github.com/typicode/husky/releases/tag/v9.0.1
> 
> But not part of this PR. We can create a JIRA ticket.

Ticket created https://jsw.ibm.com/browse/INSTA-10328